src/bootstrap/bootstrap.py

```diff
@@ -690,6 +690,10 @@ def update_submodules(self):
             recorded_submodules[data[3]] = data[2]
         for module in filtered_submodules:
             self.update_submodule(module[0], module[1], recorded_submodules)
+        polly_path = "src/llvm/tools/polly"
+        if not os.path.exists(os.path.join(self.rust_root, polly_path)):
+            run(["git", "clone", "https://github.com/llvm-mirror/polly.git",
```

Comment:
Can this not be a normal submodule placed in `src/polly` for example? I believe you can use `LLVM_EXTERNAL_POLLY_SOURCE_DIR` to tell llvm where to find it.